src/test/java/org/fundacionjala/at15/pokemon/commands/behavior/UseTest.java

```diff
@@ -31,4 +33,54 @@ public void itShouldUseMovement() throws IncompleteArguments {
 
         Assert.assertEquals(60, expectedPokemon.getHitPoints().getCurrentHitPoints());
     }
+
+    @Test
+    public void itShouldUseThrowsAnExceptionNullMoveAttack() throws IncompleteArguments {
+        try {
+            CurrentEntities current = (CurrentEntities) Reader.readJson("crt-12345678");
+            CreatePokemon params1 = new CreatePokemon();
+            new CommandLine(params1).parseArgs("-n", "pikachu", "-hp", "300");
+            params1.call();
+            current.setPokemon(params1.getIdPokemonCreated());
+            CreatePokemon params = new CreatePokemon();
+            new CommandLine(params).parseArgs("-n", "squirtle", "-hp", "100");
+            params.call();
+            current.setPokemonOpponent(params.getIdPokemonCreated());
+            current.write(new JsonWriter());
+            Use params2 = new Use();
+            params2.setMoveAttack(null);
+            params2.call();
+
+            fail();
+            } catch (IncompleteArguments ex) {
+
+            Assert.assertEquals("Error. Incomplete arguments for the Use command. Required arguments: -move.",ex.getMessage());
+        }
+    }
```

Comment:
Looks good! but I think you should delete the junk files created.
You can delete it using the function like this: 
`FileEraser.eraseFile(params.getIdPokemonCreated());`
You have 2 pokemos created by "params" and "params1"

Reply:
Done